samples/application-mobility/csm_application_mobility_v030.yaml

```diff
@@ -51,8 +50,9 @@ spec:
         image: velero/velero:v1.8.1
         imagePullPolicy: IfNotPresent
         enabled: true
-        #use-volume-snapshots: false
-        #cleanUpCRDs: false
+        use-volume-snapshots: true
```

Comment:
this should probably be "useVolumeSnapshots" just to be consistent with everything else

Reply:
done!